src/main/scala/ai/privado/tagger/config/DBConfigTagger.scala

```diff
@@ -0,0 +1,211 @@
+/*
+ * This file is part of Privado OSS.
+ *
+ * Privado is an open source static code analysis tool to discover data flows in the code.
+ * Copyright (C) 2022 Privado, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * For more information, contact support@privado.ai
+ */
+
+package ai.privado.tagger.config
+
+import ai.privado.language._
+import ai.privado.model.Constants
+import ai.privado.tagger.PrivadoSimplePass
+import ai.privado.utility.Utilities
+import ai.privado.utility.Utilities.{addRuleTags, storeForTag}
+import io.joern.dataflowengineoss.language._
+import io.joern.dataflowengineoss.queryengine.{EngineConfig, EngineContext}
+import io.shiftleft.codepropertygraph.generated.Cpg
+import io.shiftleft.codepropertygraph.generated.nodes.{CfgNode, JavaProperty}
+import io.shiftleft.semanticcpg.language._
+import overflowdb.BatchedUpdate
+import ai.privado.cache.DatabaseDetailsCache
+import ai.privado.model.DatabaseDetails
+import com.sun.jndi.dns.DnsUrl
+
+class DBConfigTagger(cpg: Cpg) extends PrivadoSimplePass(cpg) {
+
+  override def run(builder: BatchedUpdate.DiffGraphBuilder): Unit = {
+    /* println("**********************hello**********************")
+    println("**********************hello**********************")
+    println("**********************hello**********************") */
+
+    // Spring Data JDBC
+    // We are seeing duplicate values. NEED TO INVESTIGATE
+    // Let's deduplicate the properties for the time being
+    // Databases:
+    // val propertySinks = cpg.property.filter(p => p.value matches ("jdbc:.*://.*/.*|mongodb(\\+srv)?:.*")).l.groupBy(_.value).map(_._2.head)
+    val propertySinks = cpg.property.filter(p => p.value matches (".*")).l.groupBy(_.value).map(_._2.head)
```

Comment:
Do we need the filter step here?
If we look to remove the duplicates we can do `cpg.property.dedup.l`